internal/reporter/gitlab/reporter.go

```diff
@@ -88,13 +88,30 @@ func (r *reporter) SendComponentUpgrade(configCtrl internal.ConfigController,
 func (r *reporter) SendPullRequestQueue(configCtrl internal.ConfigController,
 	comp *internal.ComponentUpgradeReporter) error {
 
-	gitlabConfig, err := r.getGitlabConfig(comp.TeamName, configCtrl)
+	teamName := comp.TeamName
+	gitlabConfig, err := r.getGitlabConfig(teamName, configCtrl)
 	if err != nil {
 		return nil
 	}
 
-	repository := r.getGitlabRepository(comp, configCtrl)
-	r.overrideGitlabCredential(comp, gitlabConfig)
+	conf, err := configCtrl.Get(teamName)
+	if err != nil {
+		return nil
+	}
+	repository := ""
+	for _, b := range conf.Spec.PullRequest.Bundles {
+		if b.Name == comp.Name {
+			repository = b.Deployment.TestRunner.Gitlab.ProjectID
+		}
+	}
```

Comment:
Could we add a break, to end the loop if it founded?
```suggestion
	repository := ""
	for _, b := range conf.Spec.PullRequest.Bundles {
		if b.Name == comp.Name {
			repository = b.Deployment.TestRunner.Gitlab.ProjectID
			break
		}
	}
```